site/bps/BP-38-bypass-journal-ledger.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "BP-38: bypass journal ledger"
+issue: https://github.com/apache/bookkeeper/issues/1918
+state: "Under Discussion"
+release: "N/A"
+---
+
+### Motivation
+
+To guarantee high durability, BK write journal before flush data to persistent device which will cause two write of data.
+At the presence of replicating and auto-recovery mechanism, the two-write is a bit waste of the persistent device bandwidth,
+especially on the [scenarios](https://cwiki.apache.org/confluence/display/BOOKKEEPER/BP-14+Relax+durability) which prefer weak durability guarantee.
```

Comment:
I don't believe  our auto-recovery and replication alone can address our persistence needs. I would request you to reword it saying that - "we may not need this level of persistence under scenarios like week durability....." something like that.

Reply:
done